Bookish/Views/Catalog/Edit.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@model Bookish.Models.Book
+
+@{
+    ViewBag.Title = "Update Book";
+}
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <title>Edit Book</title>
+</head>
+<body>
+<div>
+    @using (Html.BeginForm("Edit", "Catalog", FormMethod.Post))
+    {
+        <fieldset>
+            <legend>Enter a book:</legend>
+            
+            @Html.HiddenFor(m=>m.BookID, new{value=Model.BookID})
```

Comment:
Space out these lines, e.g.
```
@Html.HiddenFor(m => m.BookID, new { value = Model.BookID })
```

Reply:
I've done this by hand but is there any shortcut to do something like this?